ksql-core/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -28,7 +28,7 @@
 
 public class KsqlConfig extends AbstractConfig {
 
-  public static final String KSQL_TIMESTAMP_COLUMN_INDEX = "ksq.timestamp.column.index";
+  public static final String KSQL_TIMESTAMP_COLUMN_INDEX = "ksql.timestamp.column.index";
```

Comment:
This bug fix is unrelated to this PR but might still be worth getting into the code.

/cc @hjafarpour @bluemonk3y .